pos_barcode_sync/README.rst

```diff
@@ -0,0 +1,54 @@
+=========================================
+ Barcode synchronization in POS
+=========================================
+
+Sync backend barcode updates for opened POSes.
+
+Credits
+=======
+
+Contributors
+------------
+* `Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>`__
+
+Sponsors
+--------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+Maintainers
+-----------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+      To get a guaranteed support you are kindly requested to purchase the module at `odoo apps store <https://apps.odoo.com/apps/modules/{VERSION}/{TECHNICAL_NAME}/>`__.
+
+      Thank you for understanding!
+
+      `IT-Projects Team <https://www.it-projects.info/team>`__
+
+Further information
+===================
+
+Demo: http://runbot.it-projects.info/demo/pos-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_barcode_sync/
+
+Usage instructions: `<doc/index.rst>`_
+
+Changelog: `<doc/changelog.rst>`_
+
+Tested on Odoo 10.0 7a63d471edec45a106fef5f783aeb7eaf43447f6
+
+Further module developing
+=========================
+
+TODO:
+-----
+
+* Price Barcodes
+* Discount Barcodes
+* Weight Barcodes
```

Comment:
Weight Barcodes are basically senseless here, because it's a part of product barcode